### DISCUSSION TITLE
enhancement: multiple linked crosshairs

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [grafana/grafana#38156] [Dashboards, Panels & Visualizations]
I am not really sure if it is worth the effort, and others will find this handy. 

If you want to compare 2 successive tests. It could be handy to create an extra crosshair that moves along with the primary crosshair and is locked to the primary crosshair by 'time distance'. Then I would be able to see eg tooltip value on both times and compare better.

I the example chart below, I would have two crosshairs at the position of the annotations. And as I move along the x-axis, I would easily see the different values of the tests.

![test](https://user-images.githubusercontent.com/13450219/94132530-b7f73080-fe5f-11ea-9ca1-4b89d711d474.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
